lib/common/MetadataCollector.ts

```diff
@@ -67,7 +67,8 @@ export class MetadataCollector implements INativeMetadataCollector {
 
   public readonly common: ICommonTagsResult = {
     track: {no: null, of: null},
-    disk: {no: null, of: null}
+    disk: {no: null, of: null},
+    movementNumber: {no: null, of: null}
```

Comment:
```suggestion
    movement: {}
```

Reply:
The nulls are inherited from the abandoned project [musicmetadata](https://github.com/leetreveil/musicmetadata), `undefined` is sufficient.